azavea/raster-vision
# Add cross-references to API docs and other tutorials in tutorial notebooks

To be rebased on #1561

## Overview

This PR makes the tutorials more useful by adding cross-references (in the form of hyperlinks) to API docs and other tutorials.

This PR also 
- makes some minor general textual and styling improvements
- fixes #1567

### Checklist


### Notes

N/A

## Testing Instructions

See https://raster-vision--1586.org.readthedocs.build/en/1586/usage/tutorials/index.html.
